### PR TITLE
Allows player slimes to use the slime faces that npc slimes can uses

### LINKED ISSUE #6882: Player slimes don't have kawaii slime faces
yes this is an issue

### PR DESCRIPTION
**Pull request by Incoming5643**
Fixes #6882

Priority: Critical

### CODE REVIEW

**Comment by Ergovisavi** [comment]
>VERB

**Comment by Incoming5643** [comment]
You can't tell me what to code at 5am you are not my mother goodnight I love you all

File: code/modules/mob/living/carbon/slime/powers.dm [hunk 1]
```diff
@@ -226,4 +226,34 @@
...
+
+	switch(selection)
+		if("cancel")
+			return
+		if("none")
+			mood = null
+		if(":>")
+			mood = "mischevous"
+		if(":3")
+			mood = ":33"
```
Comment (RemieRichards) [inline code/modules/mob/living/carbon/slime/powers.dm]: >:33

typo.

**Comment by dannno** [comment]
thank you

>VERB

yeah slimes should probably get a an actual ui instead of verbs at some point

File: code/modules/mob/living/carbon/slime/powers.dm (hunk 1 above)
Comment (Incoming5643) [inline code/modules/mob/living/carbon/slime/powers.dm]: Nope, there's two :3 icon overlays. :3, and :33. :33 looks better.

**Comment by Incoming5643** [comment]
Also I could probably code this to use /me instead of a verb, and probably will.

File: code/modules/mob/living/carbon/slime/powers.dm (hunk 1 above)
Comment (RemieRichards) [inline code/modules/mob/living/carbon/slime/powers.dm]: Hmm, scanning the code I only found :3 not :33, whatevs.

**Comment by Incoming5643** [comment]
Alright the VERB is dead :sleeping:

**Pull request merged**